packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -6,16 +6,7 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import config from '@automattic/calypso-config';
 import { getPlan, getPlanTermLabel, isFreePlanProduct } from '@automattic/calypso-products';
 import { FormInputValidation, Popover, Spinner } from '@automattic/components';
-import {
-	useSubmitTicketMutation,
-	useSubmitForumsMutation,
-	useSiteAnalysis,
-	useUserSites,
-	AnalysisReport,
-	SiteDetails,
-	HelpCenterSite,
-	useJetpackSearchAIQuery,
-} from '@automattic/data-stores';
+import { SiteDetails, HelpCenterSite } from '@automattic/data-stores';
```

Comment:
These are both types and can be moved to the `import type` import statement that imports from `@automattic/data-stores` below.